helm/templates/configmap.yaml

```diff
@@ -33,4 +33,5 @@ data:
   DATASTORE_NAME: {{ $db.name }}
   DATASTORE_TYPE: {{ $db.dbType }}
   DATASTORE_SSL_MODE: {{ $db.sslEnabled | ternary "ALLOW" "DISABLE"}}
+  SRS: {{.Values.env.srs | quote }}
```

Comment:
![image](https://github.com/user-attachments/assets/0cfd8c9f-d663-4bdb-be76-b5ba75c99c51)

Reply:
change to GEOSERVER_SRS